### DISCUSSION TITLE
Personal and public equipment list with installation manual

### DISCUSSION THREAD

**Discussion by Pighome7** [J-tour/J-tour.github.io#28] [Show and tell]
器材配置及編號已完成，請協助檢查是否有任何遺漏或需要改進的地方。

[個人器材資料及出團配置.xlsx](https://github.com/user-attachments/files/17220510/default.xlsx)

[共用器材單.xlsx](https://github.com/user-attachments/files/17220512/default.xlsx)

**Comment by ShengHaoLo** [comment]
我記得我有提供一隻麥克風，好像在 @sean880128 家

**Reply by Pighome7** [reply]
已更新至器材配置
